CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### New Features
 
+* [#60](https://github.com/clojure-emacs/inf-clojure/pull/60): Inf-clojure now supports `lumo -d` (dumb terminal) only.
```

Comment:
As there hasn't been a release that's redundant - there's already an entry about adding Lumo support.